cmd/helm/fetch.go

```diff
@@ -145,6 +146,11 @@ func (f *fetchCmd) run() error {
 		fmt.Fprintf(f.out, "Verification: %v\n", v)
 	}
 
+	var output []string
+	output = append(output, f.repoURL)
+	output = append(output, f.version)
+	fmt.Println("Download chart", strings.Join(output, ":"), "into", saved)
```

Comment:
This ends up looking like the following: 
```
$ helm fetch stable/wordpress
Download chart : into wordpress-0.4.2.tgz
```

- (small nit) `Download` might be more clear as `Downloaded` or `Downloading`
- Looks like `output` was blank so the text looks a little unclear. What do you think?


In the `--untar` scenario:
```
$ helm fetch stable/wordpress --untar
Download chart : into /var/folders/yj/tmg8tvp93sx3wv8wrtc11qq40000gn/T/helm-218381665/wordpress-0.4.2.tgz
Untar the chart into: .
```

- The path after `into` is the path to the compressed chart in the temp dir instead of the path of the uncompressed chart in my current directory.